src/utils/jsonapi.js

```diff
@@ -47,6 +48,7 @@ const serializerOptions = (serializerArgs) => {
       },
     },
     topLevelLinks: { self: topLevelSelfLink },
+    transform: transformFunction,
```

Comment:
Nice, this seems like it would be helpful. Could you add a default value for this property though?

Reply:
Spent way too much time thinking about if `false` or `null` was the better option for the default. Going with `null` because it's sufficiently false-y to stop [this check](https://github.com/SeyZ/jsonapi-serializer/blob/master/lib/serializer-utils.js#L274) and it matches better with the expected type of `transformFunction`.

Reply:
I agree `null` would be better than `false`, although I could see this causing issues if the maintainer decided to check if `opts.transform === undefined`, which is fairly common for checking if a property is on an object or not while allowing other falsey values. I think [`_.identity`](https://lodash.com/docs/4.17.15#identity) might be better here

Reply:
Hmm, if `undefined` is the more common false-y value to use here then it might make sense to leave it without a default, since that's what it ends up being if we don't give it an alternative value.

Reply:
or just use `transformFunction || undefined`, although that feels too verbose.

Reply:
I'm not sure if this is ideal either, since there are other ways to check if an object has a property like `hasOwnProperty` that would return different results if an object didn't have a property than if it had the property with value `undefined`. We should be able to make this work regardless of implementation of the jsonapi serializer package. The two main ways I can think of is to omit the property from the object that we pass in or use the identity function.

Reply:
What would the usage of `identity` look like in this case? I was looking at the docs but I'm not totally clear about what you have in mind.

Reply:
From what I understand, `_.identity` is just a name for the function `(x) => x`. So if we use the identity function as the transformation function, we're essentially saying that we aren't changing anything.

Reply:
Ah, I see what you mean. My concern there is adding the additional overhead of making an unnecessary function call for potentially every record in a large array of results, even if it is a really simple function. Although I wouldn't be surprised if the interpreter were to optimize away the function call in the case that `identity` was being used.

I think that finding a way to just not include the attribute when no `transformFunction` is passed would be a better option.

Reply:
I will look into this when I get in tomorrow 👍

Reply:
Good point. I would agree that omitting the optional property is better than explicitly setting it with something designed to not do anything.